src/gui/guiFormSpecMenu.h

```diff
@@ -388,6 +389,7 @@ class GUIFormSpecMenu : public GUIModalMenu
 	void parseListRing(parserData* data, const std::string &element);
 	void parseCheckbox(parserData* data, const std::string &element);
 	void parseImage(parserData* data, const std::string &element);
+	void parseAnimatedImage(parserData* data, const std::string &element);
```

Comment:
Old code does not always follow the coding style guidelines. Please move the pointer placement towards the variable.